client/modules/wj-css/src/components/_links.scss

```diff
@@ -0,0 +1,49 @@
+@use "sass:list";
+@use "sass:selector";
+
+a[href]:not([href*="javascript:;"]) {
```

Comment:
lol

Reply:
can't believe I had to add that. that adds at least a kilobyte to the CSS and I'm calling the police